automated-comments/csharp/general/use_expression_bodied_member.md

```diff
@@ -0,0 +1 @@
+As the `%{name}` method only has a single line of code, you could write the method as an [expression-bodied method](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members#methods).
```

Comment:
@mikedamay said:

“method only has a single line of code”. Could we say “method only has a single statement”? And would this be an appropriate place to mention that the expression can span more than one line if this aids readability?

Reply:
Good point!

> And would this be an appropriate place to mention that the expression can span more than one line if this aids readability?

Sure! I've added an example.

Absol

Reply:
Great